lua/hbac/setup.lua

```diff
@@ -2,12 +2,21 @@ local state = require("hbac.state")
 
 local M = {
 	opts = {
-		threshold = 10,
 		autoclose = true,
+		threshold = 10,
 		close_buffers_with_windows = false,
 		close_command = function(bufnr)
 			vim.api.nvim_buf_delete(bufnr, {})
 		end,
+		telescope = {
+			mappings = {
+				close_unpinned = { "<M-c>" },
+				delete_buffer = { "<M-x>" },
+				toggle_all = { "<M-a>" },
+				toggle_selections = { "<M-y>" },
```

Comment:
`<C-x>` is the Telescope default for 'Go to file selection as a split' so I hesitated to overwrite that. What about `<M-x>`?

Reply:
M-x seems reasonable